momento-sdk/src/main/java/momento/sdk/config/transport/GrpcConfiguration.java

```diff
@@ -4,32 +4,59 @@
 
 import java.time.Duration;
 import javax.annotation.Nonnull;
+import momento.sdk.internal.GrpcChannelOptions;
 
 /** Abstracts away the gRPC configuration tunables. */
 public class GrpcConfiguration {
 
   private final Duration deadline;
   private final int minNumGrpcChannels;
+  private final int maxMessageSize;
+  private final boolean keepAliveWithoutCalls;
```

Comment:
Since, in Java, there is a sane language mechanism that we can use to distinguish between whether these have been explicitly set or not, we should use it (rather than overloading the meaning of default values like `0` as we were forced to do in golang).

In Java, the right way to define these three arguments is `Optional<Boolean>`, etc.

Then for the Lambda config you can set them to `Optional.empty()`, and in the method where you're translating them to grpc, you can do `if keepaliveWithoutCalls.isSet()` instead of `> 0` etc.